py/README.md

```diff
@@ -0,0 +1,72 @@
+# Serial Keel Client for Python 3
+
+For more information on the serial-keel project see https://github.com/torsteingrindvik/serial-keel/
+
+## General structure
+
+```
```

Comment:
Add `py` or `python` behind the backticks here to let e.g. vscode properly syntax highlight this.